weed/server/raft_hashicorp.go

```diff
@@ -66,32 +114,17 @@ func NewHashicorpRaftServer(option *RaftServerOption) (*RaftServer, error) {
 	if err != nil {
 		return nil, fmt.Errorf("raft.NewRaft: %v", err)
 	}
-	if option.RaftBootstrap {
-		cfg := raft.Configuration{
-			Servers: []raft.Server{
-				{
-					Suffrage: raft.Voter,
-					ID:       c.LocalID,
-					Address:  raft.ServerAddress(s.serverAddr.ToGrpcAddress()),
-				},
-			},
-		}
-		// Add known peers to bootstrap
-		for _, peer := range option.Peers {
-			if peer == option.ServerAddr {
-				continue
-			}
-			cfg.Servers = append(cfg.Servers, raft.Server{
-				Suffrage: raft.Voter,
-				ID:       raft.ServerID(peer),
-				Address:  raft.ServerAddress(peer.ToGrpcAddress()),
-			})
-		}
+	if option.RaftBootstrap || len(s.RaftHashicorp.GetConfiguration().Configuration().Servers) == 0 {
```

Comment:
Need to wait some random time, in case all servers do this at the same time.

Reply:
A good decision would be to take a lock to the one who is the first

Reply:
add wait random time
```
I0405 10:39:21     1 raft_hashicorp.go:141] Bootstrapping idx: 0 sleep: 0s new cluster: {Servers:[{Suffrage:Voter ID:master2:9335 Address:master2:19335} {Suffrage:Voter ID:master0:9333 Address:master0:19333} {Suffrage:Voter ID:master1:9334 Address:master1:19334}]}

I0405 10:39:20     1 raft_hashicorp.go:141] Bootstrapping idx: 1 sleep: 357.129207ms new cluster: {Servers:[{Suffrage:Voter ID:master0:9333 Address:master0:19333} {Suffrage:Voter ID:master2:9335 Address:master2:19335} {Suffrage:Voter ID:master1:9334 Address:master1:19334}]}

I0405 10:39:20     1 raft_hashicorp.go:141] Bootstrapping idx: 2 sleep: 832.2263ms new cluster: {Servers:[{Suffrage:Voter ID:master1:9334 Address:master1:19334} {Suffrage:Voter ID:master2:9335 Address:master2:19335} {Suffrage:Voter ID:master0:9333 Address:master0:19333}]}
```